src/main/java/com/google/sps/servlets/utils/CommonWordsRetriever.java

```diff
@@ -14,18 +14,31 @@
 
 package com.google.sps.servlets.utils;
 
+import java.io.File;
 import java.util.ArrayList;
-import java.util.Arrays;
 import java.util.List;
+import java.util.Scanner;
 
-/** Class provides commonly used words that should be ignored within the word map */
+/** Class that provides commonly used words that should be ignored within the word map */
 public class CommonWordsRetriever {
-  // TODO: Parse text file for common words
-  private static final List<String> commonWordsList =
-      new ArrayList<>(
-          Arrays.asList(
-              "the", "it", "at", "and", "to", "we", "can", "are", "of", "is", "am", "i", "this",
-              "was"));
+  private static final List<String> commonWordsList = populateWordList();
+
+  private static List<String> populateWordList() {
```

Comment:
I think we can directly throw the error instead of try catch them

Reply:
Spoke offline about catching runtime error and benefits of not including throws clause